src/resolvers/__tests__/collection.test.js

```diff
@@ -710,6 +710,112 @@ describe('Collection', () => {
         })
       })
     })
+
+    describe('collection', () => {
```

Comment:
you don't need a new this outer level of a describe -- there is already a `describe('collection')` you can put your test in.